config.yml

```diff
@@ -0,0 +1,22 @@
+title: Simple Everviz example
```

Comment:
Remember to update the writing of the config to include these plugins as well.

https://github.com/equinor/everviz/blob/16ccc90f5675a3e7854b669d4dc7629cd0637a96/everviz/everest_hooks.py#L37

Reply:
Don't disagree, however the problem is that we don't have a source for data to add here, need to get the API ready before we can add a way to read the data.